libs/common/types.hpp

```diff
@@ -80,6 +82,17 @@ namespace iroha {
       }
       return res;
     }
+
+    static blob_t<size_> from_string(const std::string &data) {
+      if(data.size() != size_) {
+        throw BadFormatException("blob_t: input string has incorrect length");
```

Comment:
to other reviewers: this will be changed to optional as soon as Andrei will merge #552